### ISSUE TITLE
Layer_CA_06 Layer:1 00 00 00 00 00 encodes color for the preview display

### ISSUE THREAD

**Issue by jnweiger** [kkaempf/ruida#5]
Layer_CA_06 Layer:1 00 00 00 00 00    -> black 0x00 00 00
Layer_CA_06 Layer:1 00 00 00 01 7f     -> red   0x00 00 ff
Layer_CA_06 Layer:0 00 00 03 7e 00   -> green 0x00 ff 00
Layer_CA_06 Layer:1 00 00 03 7f 7f    -> yellow 0x00 ff ff
Layer_CA_06 Layer:1 00 04 00 00 40  -> dark blue 0x80 00 40
Layer_CA_06 Layer:1 00 07 7c 00 00  -> blue 0xff 00 00

cc = ((BLUE&0xff)<<16) + ((GREEN&0xff)<<8) + (RED&0xff)

**Comment by jnweiger** [comment]
Thanks @kkaempf!
I suggest that Layer_CA_06 should say Color_CA_06, now.

**Comment by kkaempf** [comment]
@jnweiger does it work at all ? I don't have an `.rd` file to test :disappointed: 
CA_06 says `Layer_Color` since 6f4bd20ac7

**Comment by jnweiger** [comment]
somewhat works.
With my favorite two layer test, I get this output for a 100% red layer 0 followed by a 100% red layer 1:
```
Layer_Color Layer:0 00 Red 00, Green 1c0, Blue 00ca 06 00 00 00 03 7e 00 
Layer_Color Layer:1 00 Red 4000, Green 00, Blue 00ca 06 01 00 00 00 01 7f 
```
[square_tri_test.rd.zip](https://github.com/kkaempf/ruida/files/1575320/square_tri_test.rd.zip)
This file is generated as a default builtin test when running `python src/rudia.py` found in https://github.com/jnweiger/ruida-laser

**Comment by kkaempf** [comment]
Fixed with 80352016c

(`+` binds higher than `>>`,`<<` in Ruby :angry: )
